src/iconsManifest/supportedExtensions.ts

```diff
@@ -1889,7 +1889,7 @@ export const extensions: IFileCollection = {
     },
     {
       icon: 'imba',
-      extensions: [],
+      extensions: ['imba','imba2'],
```

Comment:
This is not neccessary as the language id will automatically detect the extension.

Reply:
Not sure the language id works anymore as the extensions in the marketplace don't link to their repo anymore and we cannot be sure that the id is valid. To be on the safe side, I'll keep them here.